Source/FluidOscServer.cpp

```diff
@@ -291,8 +291,14 @@ void FluidOscServer::clearMidiClip(const juce::OSCMessage& message) {
 }
 
 void FluidOscServer::insertMidiNote(const juce::OSCMessage& message) {
-    if (!selectedMidiClip) return;
-    if (message.size() < 3) return;
+    if(!selectedAudioTrack){
+        std::cout << "Cannot load Audio Track: Must select Audio Track before inserting" << std::endl;
+        return;
+    }
+    if(message.size() < 3){
+        std::cout << "Expected 3 arguments, only recieved " << message.size() << "." << std::endl;
```

Comment:
How about "Cannot insert midi note: note enough arguments" That way it's clearer *which* message caused the error